openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2311,6 +2363,19 @@ This specification registers the following media types in the IANA "Media Types"
 * Change controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
 * Provisional registration? No
 
+## Uniform Resource Identifier (URI) Schemes Registry
+
+This specification registers the following URI scheme
+in the IANA "Uniform Resource Identifier (URI) Schemes" registry [@IANA.URI.Schemes].
+
+### openid-credential-offer
+
+* URI Scheme: openid-credential-offer
+* Description: Custom scheme used for credential offers
+* Status: Provisional
```

Comment:
I'm not sure I understand why this is 'Provisional' (especially given the media type registration above is not provisional)?

Reply:
I've now requested Permanent registration of the URI scheme.